src/group_theory/group_action/defs.lean

```diff
@@ -181,8 +186,6 @@ instance monoid.to_mul_action : mul_action M M :=
 This is promoted to an `add_torsor` by `add_group_is_add_torsor`. -/
```

Comment:
The priority on line 173/178 is probably pointless with the new instance - that or we should add it on the new one too. Is that priority on semiring.to_module?

Reply:
Oh it is the same as `semiring.to_module`: https://github.com/leanprover-community/mathlib/blob/e8add823aff3bbd5025670973767539936f98e6a/src/algebra/module/basic.lean#L235

I'm happy to add the same priority on the new instance and reluctant to tinker with what is in place.